packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -68,17 +75,19 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
                 }))
               : [];
           }
-
-          const styleProps: IExampleCardStyleProps = { isRightAligned, isScrollable, isCodeVisible, theme };
+          const styleProps: IExampleCardStyleProps = { isRightAligned, isScrollable, theme, isCodeVisible };
           const classNames = (this._classNames = getClassNames(styles, styleProps));
           const { subComponentStyles } = classNames;
           const { codeButtons: codeButtonStyles } = subComponentStyles;
 
-          const exampleCardContent = (
-            <div className={classNames.example} data-is-scrollable={isScrollable}>
-              {children}
-            </div>
-          );
+          const exampleCardContent =
+            this.props.isCodeVisible && this.canRenderLiveEditor ? (
+              <EditorPreview id={this.props.title.replace(' ', '')} />
```

Comment:
You should probably apply `classNames.example` to this too (just add a `className` prop to the component if it doesn't already have one)

Reply:
Also wonder if it will need `data-is-scrollable` like the normal example div (not sure exactly where that's being used)

Reply:
Should I add `data-is-scrollable` just in case?

Reply:
I think so, though to actually get it applied you might have to add a getNativeProps call in EditorPreview.